backend/app/routers/areas.py

```
@@ -0,0 +1,107 @@
+from app.schemas import AreaCreate, AreaOut, AreaUpdate
+from app.utils import area_to_dict, make_id
+
+router = APIRouter(
+    prefix="/api/areas",
+    tags=["areas"],
+    dependencies=[Depends(require_admin)],
+)
+
+
+@router.post("", response_model=AreaOut, status_code=status.HTTP_201_CREATED)
+async def create_area(body: AreaCreate, db: AsyncSession = Depends(get_db)) -> dict:
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)